spec/beaker/dsl/helpers/web_helpers_spec.rb

```diff
@@ -36,21 +36,43 @@ def fetch_allows
       it "returns its second and third arguments concatenated." do
         concat_path = "#{destdir}/#{name}"
         create_files([concat_path])
-        expect( logger ).to receive( :notify ).with( /^Already\ fetched\ / )
```

Comment:
The behavior flipped, so "Already fetched" changed code paths to the "Fetching and saving to" notifications.

This is not what this spec test is saying it tests, however, so I stopped it from testing that, since we do have tests for it below.